src/discord-cluster-manager/utils.py

```diff
@@ -107,17 +134,6 @@ async def send_logs(thread: discord.Thread, logs: str) -> None:
         await thread.send(f"```\n{chunk_text}\n```")
 
 
-def extract_score(score_str: str) -> float:
```

Comment:
do we no longer need this?

Reply:
no, haven't needed this for a long time. this was for parsing stdout to try and  find a substring that looked like it wanted to report a score